Dockerfile

```diff
@@ -1,8 +1,35 @@
-FROM php:8.2.0RC4-zts-bullseye AS php-base
-
-# Note that this image is based on the official PHP image, once 8.3 is released, this stage can likely be removed
-
-RUN rm -Rf /usr/local/include/php/ /usr/local/lib/libphp.* /usr/local/lib/php/ /usr/local/php/
+# syntax=docker/dockerfile:1.4
+
+ARG DISTRO
+ARG PHP_VERSION
```

Comment:
It'd be a good idea to update the documentation on how to build a one-off image (especially for testing the build) as Docker Bake is quite slow.

Reply:
Official images never use build args because they pollute history. I would like to do the same here, please.

Reply:
I need to diff changes and get them here.